ruby/vampires.rb

```diff
@@ -0,0 +1,125 @@
+def age_calculated (year_born)
+  current_year = Time.now.year
+  return current_year - year_born
```

Comment:
Ruby is an implicit language meaning that it will implicitly return the last line it runs in a method.  Explicit returns are great to use in the proper setting but in this case leaving `return` out works just fine.